tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -196,9 +196,9 @@ def montecarlo_main_loop(
         )
 
     # Configuring the Tracking for R_Packets
-    tracked_rpackets = List()
+    rpacket_trackers = List()
     for i in range(len(output_nus)):
-        tracked_rpackets.append(RPacketCollection())
+        rpacket_trackers.append(RPacketTracker())
```

Comment:
```suggestion
        rpacket_trackers = [RPacketTracker() for i in range(len(output_nus))]
```

Reply:
@wkerzendorf This is not feasible as the function under consideration is a `NJIT` function. List comprehension doesn't work under `NJIT` function as type of the object is undefined.